samples/hssi/hssi_10g_cmd.h

```diff
@@ -159,8 +170,13 @@ class hssi_10g_cmd : public hssi_cmd
     if (eth_ifc_ == "none")
         eth_ifc = hafu->ethernet_interface();
 
+    if (port_ != INVALID_PORT)
```

Comment:
corner case:  if user pass all port_, dst_port_,  src_port_ values  HSSI tool overrides  dst_port_,  src_port_ values.

Reply:
Yes, would you like me to also print a message in this case?

Reply:
Print message and continue